build.xml

```diff
@@ -317,7 +317,7 @@
     <ecj srcdir="${aosp.src.dir}" destdir="${aosp.classes.dir}" gensrcdir="${aosp.gensrc.dir}">
       <ecj-params>
         <!-- ManifestMerger.java has redundant null checks -->
-        <compilerarg value="-err:-null" />
```

Comment:
There is a ton of new warnings in `ManifestMerger2` (73!) that ant turns into errors